preup/kickstart/application.py

```diff
@@ -210,10 +210,28 @@ def comment_kickstart_issues(self):
                 kickstart_data[index] = "#" + row
         FileHelper.write_to_file(self.kick_start_name, 'wb', kickstart_data)
 
+    def _get_all_postinstall_scripts(self):
+        found_scripts = []
+        for (path, dirs, files) in os.walk(os.path.join(settings.result_dir, settings.kickstart_dir)):
+            for f in files:
+                file_name = os.path.join(path, f)
+                if os.path.exists(file_name) and FileHelper.check_file(file_name, u"x") is True:
+                    found_scripts.append(file_name)
+        return found_scripts
+
     def generate(self):
         if not self.collect_data():
             log_message("Important data are missing for the Kickstart generation.", level=logging.ERROR)
             return None
+        if not self._get_all_postinstall_scripts():
+            accept = ['y', 'yes']
+            log_message("You did not specify any postinstall scripts. "
+                        "Include them to %s " % os.path.join(settings.result_dir, settings.postinstall_dir))
```

Comment:
*"You did not specify any postinstall scripts. Include them to .."* sounds like the user should have done that. However this step is optional.
I would rephrase it to *The %s folder is empty - scripts to be executed after the migration should be placed here. ... Do you want to continue with the kickstart generation without any postmigration scripts?"